>From 393c36efb1a54c29a15aae1a2ba95cc4e189da35 Mon Sep 17 00:00:00 2001
Message-Id: <393c36efb1a54c29a15aae1a2ba95cc4e189da35.1486026065.git.hahn@univention.de>
From: Philipp Hahn <hahn@univention.de>
Date: Thu, 2 Feb 2017 09:58:42 +0100
Subject: [PATCH] man: dh_python2 --namespace requires argument
Organization: Univention GmbH, Bremen, Germany

Update the manual page to make it clear, that the option --namespace
of dh_python2 requires an argument:
> parser.add_option('--namespace', action='append', dest='namespaces',
>                   help='recreate __init__.py files for given namespaces at install time')
---
 dh_python2.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dh_python2.rst b/dh_python2.rst
index 593ad09..971f96d 100644
--- a/dh_python2.rst
+++ b/dh_python2.rst
@@ -197,7 +197,7 @@ OPTIONS
 --requires=FILENAME	translate requirements from given file(s) into Debian
   dependencies and add them to ${python:Depends}
 
---namespace	use this option (multiple time if necessary) if
+--namespace=NAMESPACE	use this option (multiple time if necessary) if
   namespace_packages.txt is not complete
 
 --ignore-namespace	ignore Egg's namespace declaration and
-- 
2.1.4

